guppylang/std/qsystem/__init__.py

```diff
@@ -17,6 +17,19 @@
 @guppy
 @no_type_check
 def phased_x(q: qubit, angle1: angle, angle2: angle) -> None:
+    r"""phased_x gate command.
+
+    .. math::
+
+        \mathrm{phased\_x}(q, \theta_1, \theta_2)=
```

Comment:
I'm not sure we need the qubits in the argument lists here. They don't appear on the RHS so seems a bit odd to have them on the LHS.

I suggest using pytket capitalizations everywhere, e.g. `\mathrm{PhasedX}(\theta_1, \theta_2)`.

Reply:
The qubit arguments were a deliberate choice here (not sure if it was the optimal choice)

Unlike pytket, Guppy has no notion of a `CX` gate as an `Op` independent of qubits. Guppy implements gates as functions which take qubits as arguements (more like a `Command` in pytket. It seemed more correct to add them.

This is potentially hairy and unsatisfying though as the matrix defintion could vary in cases where the qubits passed as arguments are not adjacent or are passed in a different order.

Do you have a preference @ss2165?

Reply:
I think you have to separate the matrix notation from the function call notation. So no qubits here, and state somewhere that qubit ordering is implicitly always in the order they appear in the function arg list, or introduce some extra notation 

```markdown
rzz(q1, q2, theta_2)
Qubit ordering: [q1, q2]
.. math::
    \mathrm{RZZ}(\theta_2} = 
```

Reply:
Added in [109f294](https://github.com/CQCL/guppylang/pull/912/commits/109f294d1b19df84d27101ff3472d4bd1c02e681) and [7c730be](https://github.com/CQCL/guppylang/pull/912/commits/7c730be9f4a3764a7e0b25c2d3d2c0f09a375dad)